paper/paper.md

```diff
@@ -77,16 +77,11 @@ are not accessible using existing codes.
 When `BurnMan` was first released [@Cottaar:2014], its focus was on
 the seismic properties of the lower mantle, using a single endmember mineral
 database [@Stixrude:2011] as a foundation. Since then,
-its scope has expanded considerably. Some workflows of the current code
-are provided in \autoref{fig:workflows}. 
-
-![Some possible BurnMan workflows. Input/output shown as ellipses, functions as
-rectangles and classes as diamonds.\label{fig:workflows}](figures/workflows.png)
-
+its scope has expanded considerably. 
 `BurnMan` now contains equations of state
 for minerals and melts from several published datasets. A model for anisotropy
 has recently been added. Several solution model formulations are included
-(\autoref{fig:garnetsolution}),
+(with one example model shown in \autoref{fig:garnetsolution}),
```

Comment:
Because the text says “Several solution model formulations are included (Figure 2)” but then Figure 2 only shows “a published model”.